Elevation_Projects/My_Roster_EJN/Dist/main.js

```diff
@@ -0,0 +1,22 @@
+const onLoad = function () {
```

Comment:
try to be consistent with the way you write your functions. If you like arrow functions, use arrow functions